src/generators/notebook/options_manager.ts

```diff
@@ -75,7 +75,7 @@ class OptionsManager extends Registry.IOptionsManager {
    */
   set notebookMetadata(value: [string, any]) {
     if (this._notebook.currentWidget != null) {
-      this._notebook.currentWidget.model.metadata.set(value[0], value[1]);
+      this._notebook.currentWidget.model!.metadata.set(value[0], value[1]);
```

Comment:
Here and below: rather than asserting the model exists with `!`, can we check it in the if statement instead?

Reply:
We can do that instead if you think it's better, I did it this way to follow how it was done in JupyterLab. The change that caused this issue was this pr https://github.com/jupyterlab/jupyterlab/pull/7657 which used non-null asserts for the same values here where they're accessed inside jupyterlab code

Reply:
Hmm, I think the fact that the notebook panel is typed as possibly-null is a bit of historical cruft, I don't know that there is much model-swapping going on right now.

That being said, I'm a fan of strict null checks, and would rather follow them over using the non-null assertion. It's not a huge deal either way.

Reply:
I am also a fan of strict null checks, but tend to prioritize consistency over preference when a methodology seem to have been established.

Reply:
Hi I am new to git hub. Does the above mean, the fix is done and the toc now can work for jupyter lab 2.0. I tried to install but build error. Could someone advise when this can be ready, because without TOC, i am lost in the notebook.